site/src/css/global/table.css

```diff
@@ -1,44 +1,44 @@
-table {
+.docs-page-content table {
```

Comment:
Could we move these styles to `css/docs` since they're not global anymore please